### PR TITLE
CI/DOCS: Fix more `CI/` paths

### PR DESCRIPTION
**Pull request by mjansenDatabay**
Hi folks,

this PR fixes more paths in our CI tool configurations and our documentation.

Best regards,
Michael

### CODE REVIEW

File: components/ILIAS/WOPI/resources/js/rollup.config.js
Comment (mjansenDatabay) [inline components/ILIAS/WOPI/resources/js/rollup.config.js:17]: @chfsx This is a productively used file (for developers, means: more than CI and documentation), please have a short look here.

**Review by thibsy — APPROVED** [review]
Hi @mjansenDatabay

Thx for fixing all remaining usages of the legacy path `CI/`. I have compared these changes with a local regex-search and came to the same amount of usages. We should have got all of them now :).

LGTM (changes to UI doc as well), I will merge this after @chfsx approved for WOPI.

Kind regards,
@thibsy

**Review by chfsx — APPROVED** [review]
LGTM, thank you very much @mjansenDatabay

**Pull request merged**
